models/task/task.go

```diff
@@ -264,7 +272,7 @@ func (u *TaskStruct) ChangeList(listId int64, prevToTaskId *int64, repo *reposit
 			issueAction = &listOption.Action
 		}
 		// issueを移動
-		result, err := hub.EditGithubIssue(token, repo, u.IssueNumber.Int64, []string{*label.Name}, &u.Title, &u.Description, issueAction)
+		result, err := hub.EditGithubIssue(token, repo, u.IssueNumber.Int64, labelName, &u.Title, &u.Description, issueAction)
```

Comment:
__[gometalinter]__
* [structcheck] unused struct field cannot pass argument repo (variable of type *../repository.RepositoryStruct) to parameter of type *../../models/repository.RepositoryStruct
* [gotype] undeclared name: hub
* [vetshadow] declaration of err shadows declaration at task.go:229: 
* [aligncheck] cannot pass argument repo (variable of type *../repository.RepositoryStruct) to parameter of type *../../models/repository.RepositoryStruct